esp/src/src/nls/bs/hpcc.ts

```diff
@@ -125,8 +125,10 @@
     ContinueWorking: "Nastavite sa radom",
     Copied: "Kopirano!",
     Copy: "Kopirajte",
+    CopyLogicalFiles: "Kopirajte logičke fajlove u privremeni spremnik",
```

Comment:
Please replace the above with the "Kopirajte logičke fajlove u klipbord" to make it consistent with other content

Reply:
change made